android/androiddebug.cpp

```diff
@@ -22,60 +22,60 @@
  *
 \******************************************************************************/
 
-const char*const applicationName = "Jamulus";
+const char* const applicationName = "Jamulus";
 
 #ifdef ANDROIDDEBUG // Set in my myapp.pro file for android builds
-#include <android/log.h>
-#include <QString>
-#include <QEvent>
-#include <QDebug>
-#include <stdio.h>
-#include <math.h>
-#include <string>
+#    include <android/log.h>
+#    include <QString>
+#    include <QEvent>
+#    include <QDebug>
+#    include <stdio.h>
+#    include <math.h>
+#    include <string>
```

Comment:
Is this expected? It looks strange to me having `#` separated from `include`.

Reply:
> Is this expected? It looks strange to me having `#` separated from `include`.

Yes, I think it was the decision to do that when discussing the options (no indentation at all, indent with `#`, indent but keep `#` on first column). I wanted to link the discussion here, but I suspect it's hidden in one of the conversations as part of this PR.

Such things could be changed again in a rather simple way later, if we decide otherwise. I think we should really focus on getting this long-term, huge PR merged. It will not be the last one. :)

Reply:
Found one part of the conversation here, for reference: https://github.com/jamulussoftware/jamulus/pull/1127#discussion_r606452340